packages/db/src/index.ts

```diff
@@ -1,7 +1,9 @@
-import type { DrizzleConfig } from "drizzle-orm";
+import * as drizzleHelpers from "drizzle-orm";
 import { drizzle } from "drizzle-orm/postgres-js";
 
 export const database = <TSchema extends Record<string, unknown>>(
   url: string,
-  params?: DrizzleConfig<TSchema>,
+  params?: drizzleHelpers.DrizzleConfig<TSchema>,
 ) => drizzle({ connection: { url, max: 1 }, ...params });
+
+export const { eq, notExists, sql } = drizzleHelpers;
```

Comment:
This is a fragile and somewhat arbitrary way to gain access to these primitives. A wildcard import might be best.

Better yet, read the `packages/db/src/drizzle.ts` file. There may be a better way involving that file to gain access to Drizzle functions.